package.json

```diff
@@ -71,6 +71,7 @@
     "build": "wp-scripts build",
     "lint": "npm-run-all --parallel lint:*",
     "lint:js": "wp-scripts lint-js js",
+    "lint:js:files": "wp-scripts lint-js",
```

Comment:
This command allows passing specific files, like `npm run lint:js:files js/foo.js`. 

`npm run lint:js` lints all of the JS files in `js/`.

Reply:
n and p to navigate between commits in a pull request.